gems/pending/disk/modules/MSCommon.rb

```diff
@@ -219,134 +219,136 @@ def self.getFooter(file, skip_check = false)
     footer
   end
 
-  private
-
-  def self.getHeader(footer, skip_check = false)
-    hdrLoc = getHiLo(footer, "data_offset")
-    hdrSiz = HEADER.size
-    puts "VHD Header is mislocated: 0x#{'%04x' % hdrLoc} (s/b 0x0200)" if hdrLoc != HEADER_LOCATION && !skip_check
-    @file.seek(hdrLoc, IO::SEEK_SET)
-    buf = @file.read(hdrSiz)
-    header = HEADER.decode(buf)
-    # TODO: Find out why this checksum test is failing.  For now don't call getHeader without skip_check set to "true"
-    unless skip_check
-      headerCsum = checksum(buf, 36)
-      raise "Header checksum doesn't match: got 0x#{'%04x' % headerCsum}, s/b 0x#{'%04x' % @header['checksum']}" if headerCsum != header['checksum']
+  class << self
```

Comment:
@AndyStabler thanks for going through all of these... for this one... we prefer using private_class_method over `class << self` with a `private` section for readability.  It's easy to keep adding to the singleton class and not realizing what context you're in, especially when there's so many pages of singleton methods.  Feel free to either do this file or skip it.  We can finish up the annoying ones.  Your contribution is greatly appreciated!